.env

```
@@ -0,0 +1,4 @@
+DB_HOST=localhost
+DB_USER=treby
+DB_PASSWORD=password1234
+DB_NAME=todo_app
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)